### PR TITLE
BH-16 - Create Comment Model

### PR DESCRIPTION
**Pull request by brunodeoliveira1**

### CODE REVIEW

**Comment by robecilla** [comment]
A couple of things:
- What's that commit message? We need to follow the following format: <PROJECT-KEY> - meaningfull commit message 
- The name of the PR needs to have the project key and the name of the ticket. E.g: BH-16 - Log comments against registrations.
- The ticket it's just about models but I see more files changed. We can squeeze this in the same ticket but we need to remove the other tickets then.
I would also recommend the practice of "atomic commits". Commit pieces of software that make sense.

Until this point are solved, we can not merge this.

File: api/models.py [hunk 1]
```diff
@@ -24,3 +24,10 @@ class Registration(models.Model):
 
     def __str__(self):
         return f'{self.date} {self.project.name}'
+
+class Comments(models.Model):
+    comments = models.CharField(max_length=1000, unique=True)
```
Comment (robecilla) [inline api/models.py]: I would rename comments for content. This field is about comment contents.

File: api/models.py [hunk 2]
```diff
@@ -24,3 +24,10 @@ class Registration(models.Model):
...
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
```
Comment (robecilla) [inline api/models.py]: This is not correct. The class Comment does not have name property. I would't include this method for this case.

File: api/serializers.py [hunk 3]
```diff
@@ -17,3 +17,8 @@ class ReportingTypeSerializer(serializers.ModelSerializer):
     class Meta:
         model = ReportingType
         fields = ('id', 'name')
+
+class CommentsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ReportingType
```
Comment (robecilla) [inline api/serializers.py]: This is not correct. You need the Comment model here.

File: api/models.py [hunk 4]
```diff
@@ -24,3 +24,10 @@ class Registration(models.Model):
 
     def __str__(self):
         return f'{self.date} {self.project.name}'
+
+class Comments(models.Model):
```
Comment (robecilla) [inline api/models.py]: Model classes are better understood when they are singular. Change this for Comment

File: api/models.py [hunk 5]
```diff
@@ -24,3 +24,10 @@ class Registration(models.Model):
...
+    comments = models.CharField(max_length=1000, unique=True)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
```
Comment (robecilla) [inline api/models.py]: This does not make sense. Comments are made on Registrations, not on Projects

File: api/serializers.py (hunk 3 above)
Comment (brunodeoliveira1) [reply api/serializers.py]: to take care in another ticket.

File: api/models.py (hunk 5 above)
Comment (brunodeoliveira1) [reply api/models.py]: done

**Pull request merged**